scipy/stats/tests/test_hypotests.py

```diff
@@ -1004,6 +1006,186 @@ def test_less_greater(self, input_sample, expected, alternative):
         )
 
 
+class TestBoschlooExact:
+    """Some tests to show that boschloo_exact() works correctly."""
+
+    @pytest.mark.parametrize(
+        "input_sample,expected",
+        [
+            ([[43, 40], [10, 39]], (0.9999374, 0.9998659)),
+            ([[2, 7], [8, 2]], (0.01852173, 0.009886142)),
+            ([[5, 1], [10, 10]], (0.9782609, 0.9450994)),
+            ([[5, 16], [20, 25]], (0.08913823, 0.05827348)),
+            ([[5, 0], [1, 4]], (1, 1)),
+            pytest.param(
+                [[0, 1], [3, 2]], (0.5, 0.34375),
+                marks=pytest.mark.xfail_on_32bit(
+                    "This test is sensitive to machine epsilon level",
+                    reason="numerical error"),
+            ),
+            ([[0, 2], [6, 4]], (0.2272727, 0.09850981)),
+            ([[2, 7], [8, 2]], (0.01852173, 0.009886142)),
+            ([[7, 12], [8, 3]], (0.06406797, 0.03410916)),
+            ([[10, 24], [25, 37]], (0.2009359, 0.1512882)),
+        ],
+    )
+    def test_less(self, input_sample, expected):
+        """The expected values have been generated by R, using a resolution
+        for the nuisance parameter of 1e-8 :
+        ```R
+        library(Exact)
+        options(digits=10)
+        data <- matrix(c(43, 10, 40, 39), 2, 2, byrow=TRUE)
+        a = exact.test(data, method="Boschloo", alternative="less",
+                       tsmethod="central", np.interval=TRUE, beta=1e-8)
+        ```
+        """
+        res = boschloo_exact(input_sample, alternative="less")
+        statistic, pvalue = res.statistic, res.pvalue
+        assert_allclose([statistic, pvalue], expected, atol=1e-4)
+
+    @pytest.mark.parametrize(
+        "input_sample,expected",
```

Comment:
The p-values are not precisely those that I get when using the code above. How were these numbers generated?

Reply:
Maybe I should have been clearer in this documentation string. The `exact.text` function from [exact package](https://rdrr.io/cran/Exact/man/exact.test.html) doesn't take the table the same way as us. For example, let's take the first table we are testing : `[[43, 40], [10, 39]]`.
In my function, we would do : 
```python
stats.boschloo_exact([[43, 40], [10, 39]])
```
whereas in r we would do : 
```r
data <- matrix(c(43, 10, 40, 39), 2, 2, byrow=TRUE)
a = exact.test(data, method="Boschloo", alternative="greater", tsmethod="two.sided", np.interval=TRUE, beta=1e-8)
```
Basically, `exact.test` is taking the two elements from the first column first (43, 10) and then the two elements from the second column (40, 39).
Using this method, do you find the expected pvalue ? 
(maybe my explanation is not necessary, and therefore I apologize for the long comment)